### PR TITLE
fix: App URL

### PR DESCRIPTION
**Pull request by gbarkhatov**
- Layout OG image uses network config instead of predefined link
- Same with Terms

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -29,14 +31,14 @@ export default function RootLayout({
       <meta property="og:image:height" content="1170" />
       <meta
         property="og:image"
-        content="https://btcstaking.babylonchain.io/og.png"
+        content={`${getNetworkConfig().appUrl}/og.png`}
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/layout.tsx:34]: @hiepmai-babylonchain ?

[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/layout.tsx:34]: @maiquanghiep

File: src/config/network.config.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ interface NetworkConfig {
   networkName: string;
   mempoolApiUrl: string;
   network: Network;
+  appUrl: string;
```
[the remark this answers is not in the document]
Comment (vitsalis) [reply src/config/network.config.ts:12]: Agree with the comment here. Regarding the above options, let's use (2).

File: src/app/layout.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (maiquanghiep) [reply src/app/layout.tsx:34]: We are waiting for the decision on testnet domain migration. https://babylonlabsworkspace.slack.com/archives/C07DYV8TEDD/p1722937105585269?thread_ts=1722832028.082259&cid=C07DYV8TEDD
The testnet domain is still `https://btcstaking.testnet.babylonchain.io/` so there are no images on babylonlabs domain yet

File: src/config/network.config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/config/network.config.ts:12]: @vitsalis @jrwbabylonlab 

Added `getNetworkAppUrl` function close to `shouldDisplayTestingMsg` (`src/config/index.ts`)

`NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES`:

- `true`: https://i.imgur.com/xdlgYOB.png
- `false`: https://i.imgur.com/CvvBlIJ.png

After we have `https://btcstaking.babylonlabs.io`, we can easily change it in one place

**Review by jeremy-babylonlabs — APPROVED** [review]

**Pull request merged**